src/Collection/AuditedCollection.php

```diff
@@ -75,12 +75,24 @@ class AuditedCollection implements Collection
      * - store entity
      * - contain audited entity.
      *
-     * @var Collection<int|string, object|array<string, mixed>>
+     * @var Collection<int|string, array>
      *
-     * @phpstan-var Collection<TKey, T|array{keys: array, rev: string|int}>
+     * @phpstan-var Collection<TKey, array{keys: array, rev: string|int}>
      */
     protected $entities;
 
+    /**
+     * Entity collection. If can be:
+     * - empty, if the collection has not been initialized yet
+     * - store entity
+     * - contain audited entity.
```

Comment:
Maybe this comment is not right anymore? Both properties has the same , but you kinda splitted how the previous property works, right?

Reply:
I tried to improve the comment